### PR TITLE
Pfeffer Dust-off

### PR DESCRIPTION
**Pull request by acpaquette**
Updated ale version and made some small changes to get tests passing

### CODE REVIEW

**Comment by acpaquette** [comment]
Currently ones test fails because the fix for it is a PR in ALE

File: pfeffernusse/openapi/openapi.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ paths:
       summary: Converts Image Labels to ISDs
       x-swagger-router-controller: pfeffernusse.controllers.default_controller
       x-openapi-router-controller: pfeffernusse.controllers.default_controller
-  /metalkernels/:
```
Comment (AustinSanders) [inline pfeffernusse/openapi/openapi.yaml:40]: 🤘

File: pfeffernusse/controllers/default_controller.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import connexion
 import six
 
+from pfeffernusse import util
```
Comment (AustinSanders) [inline pfeffernusse/controllers/default_controller.py:6]: Are we using pfeffernusse's util for anything?

Comment (acpaquette) [reply pfeffernusse/controllers/default_controller.py:6]: Nope! I imported it to compare it to how ALE was getting the metakernels. I'll remove the import

**Pull request merged**